gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/rest/api/service/sanitizer/UrlSanitizerUtilsTest.java

```diff
@@ -17,12 +17,15 @@
 
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 import io.gravitee.rest.api.service.exceptions.InvalidDataException;
 import io.gravitee.rest.api.service.exceptions.UrlForbiddenException;
 import java.util.Collections;
 import org.apache.commons.text.RandomStringGenerator;
 import org.junit.Test;
+import org.junit.jupiter.params.ParameterizedTest;
```

Comment:
Could you also migrate all tests to use Junit Jupiter everywhere and avoid Junit4 assertions?

Reply:
done :)